### ISSUE TITLE
Terms & Conditions Re-Structure

### ISSUE THREAD

**Issue by chris206** [206Tours/DB#1605]
Hi Brad,

After discussing this with Sandy -

We need the ability to create "Term Sets" that are hosted externally from each tour. Modification in one place would revise what clients see in all places that have selected the same Terms set.

It seems that the DB is/was able to do something like this

We need the ability to create these "Term Sets" which are drop-down selectable in ALL TOURS.
One drop down can be "Custom" and it would allow a manual override originating and reflecting ONLY within the single tour where it was selected. Users would then be able to manually type the terms here.

![image](https://cloud.githubusercontent.com/assets/22382144/19820885/621c4e98-9d29-11e6-8bc4-3be10021c7b2.png)

I know that this will be difficult for old departures - but it will avoid similar terms correction issued in the future

**Comment by bradatkins** [comment]
This is now complete.

**Comment by chris206** [comment]
awesome!
Where is the list of various "term sets" located?

**Comment by bradatkins** [comment]
Check the email I just resent

**Comment by chris206** [comment]
I'm totally clueless on this one

**Comment by chris206** [comment]
I'm still lost on this revision, please help me understand. Does this fix alter terms - not just in the DB - but also on invoice?
as per liz

![image](https://cloud.githubusercontent.com/assets/22382144/22555193/e58255e0-e930-11e6-829a-e55a1b351b95.png)

**Comment by chris206** [comment]
Hey Brad,

The fix here is non-intuitive. There needs to be a single place that we can modify ALL terms, and then those terms need to be selectable but not editable from a drop down in ALL TOURS.

This goes to our "Change Terms for 1000 Tours in 1 place" strategy. If I update "STANDARD TERMS"in a single location, any departure with "STANDARD TERMS" will have its terms updated.

The way your fix works still has tours individually located in each tour. This is imperative to protect the company and our pilgrims.

Once this is complete, we can work on how the terms interacts with initial confirmation email - currently the info contradicts itself and obeys no editable standard.

![image](https://cloud.githubusercontent.com/assets/22382144/24968276/c88fa8b8-1f7a-11e7-869b-f62eb146ef9a.png)

![image](https://cloud.githubusercontent.com/assets/22382144/24968297/d53912c0-1f7a-11e7-96da-b650e61f23b8.png)

![image](https://cloud.githubusercontent.com/assets/22382144/24968307/dc7739b8-1f7a-11e7-9e9d-88f8895d3d92.png)

**Comment by chris206** [comment]
Where are the "standard terms" actually located?

I've deleted them in tour, and they re-appear when i open it back up. What is the modifiable source of this info?

**Comment by chris206** [comment]
Where are the "standard terms" actually located?

I've deleted them in tour, and they re-appear when i open it back up. What is the modifiable source of this info?

It seems that this is already done, but missing a Master Location to modify the body of each Terms Set
Can you just structure it differently so that Terms title and body are accessible in the same place

**Comment by bradatkins** [comment]
They are in a seperate text file, as Ronen uses them for the Website as well

**Comment by chris206** [comment]
Can that be accessible from a single location to Admin only?
Where is the text file for TEST?
